x-pack/server/lib/create_router/call_with_request_factory/call_with_request_factory.js

```diff
@@ -6,13 +6,12 @@
 
 import { once } from 'lodash';
 
-const callWithRequest = once(server => {
-  const cluster = server.plugins.elasticsearch.getCluster('data');
-  return cluster.callWithRequest;
-});
-
 export const callWithRequestFactory = (server, request) => {
   return (...args) => {
-    return callWithRequest(server)(request, ...args);
+    const callWithRequest = server => {
```

Comment:
this also wasn't necessarily an issue, but since this is now common code shared by multiple plugins, it should be a true factory